docs/challengeSolutions.md

```diff
@@ -38,21 +38,66 @@ The above challenge was completed using Burp Suite Community Edition.
 
 ## Broken User Authentication
 
-### Challenge 3 - Reset the password of a different user
+### [Challenge 3 - Reset the password of a different user](challenges.md#challenge-3---Reset-the-password-of-a-different-user)
+
+#### Detailed solution
+
+1. Go to the login page http://localhost:8888/login and click on forgot password and you will be brought to http://127.0.0.1:8888/forgot-password.
```

Comment:
```suggestion
1. Go to the login page {baseURL}/login and click on forgot password and you will be brought to http://127.0.0.1:8888/forgot-password.
```
The base URL changes as per deployment, it would be better to make this change everywhere.